tests/fixtures/collection/tox.ini

```diff
@@ -2,6 +2,10 @@
 skipdist = true
 envlist = lint_all
 
+[ansible]
+molecule_opts =
+  -c {toxinidir}/molecule.yml
```

Comment:
We also need to test if the syntax looks like this:

```ini
molecule_opts =
  -c
  {toxinidir}/molecule.yml
```

I'm not sure your code above supports that. If we don't want to support all ways this could be parsed we could instead force people to just list config files like we do for yamllint_config so they would then do

```ini
molecule_config_files =
  {toxinidir}/molecule_one.yml
  {toxinidir}/molecule_two.yml
```

And so forth.

Reply:
I confirm the current implementation does not manage this case. I'll handle it like how it actually works for yamllint_config and ansible_lint_config. It'll make everything uniform and compliant.